language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/TypedBindingPatternNodeContext.java

```diff
@@ -49,19 +49,20 @@ public TypedBindingPatternNodeContext() {
         associated with the type descriptor. Otherwise the router will go up the parent ladder.
          */
         if (withinTypeDesc(context, node) || node.typeDescriptor().kind() == SyntaxKind.TABLE_TYPE_DESC
+                || node.typeDescriptor().kind() == SyntaxKind.FUNCTION_TYPE_DESC
```

Comment:
Any sample for the newly added case here?

Reply:
> Any sample for the newly added case here?

https://github.com/ballerina-platform/ballerina-lang/pull/31226/files/83c786057b65ce4c564471fc4b70c7bd1dd4520a#diff-b15d7c54759a52b7063c5f52dd4b769ce52ad3ea7dcd7dfde8da8debe8a331c4